### PR TITLE
enable and render koji_parent plugin

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
Change the commit message to say koji_import? ;-)

**Comment by twaugh** [comment]
Looks great except for commit message and PR title.

**Comment by lcarva** [comment]
> Looks great except for commit message and PR title.

Good catch! Fixed it.

**Comment by lcarva** [comment]
Important: This plugin must be disabled for base images. Do not merge until that's addressed.

**Comment by lcarva** [comment]
> Important: This plugin must be disabled for base images. Do not merge until that's addressed.

This is now handled.

**Comment by lcarva** [comment]
> Please add tests to tests/build_/test_arrangements.py.

Thanks! Added it.

**Comment by twaugh** [comment]
Could mention this additional difference v1->v2 in the docstring for TestArrangementV2.

File: tests/build_/test_arrangements.py [hunk 1]
```diff
@@ -428,3 +429,32 @@ def test_add_filesystem_in_worker(self, osbs, base_image, scratch,
...
+        (True, 'koji/image-build', False),
+        (True, 'foo', False),
+        (False, 'koji/image-build', False),
+        (False, 'foo', True),
+    ])
+    def test_koji_parent_in_orchestrator(self, osbs, base_image, scratch,
+                                         expect_plugin):
+        additional_params = {
+            'base_image': base_image,
+            'yum_repourls': ['https://example.com/my.repo'],
```
Comment (twaugh) [inline tests/build_/test_arrangements.py]: Is 'yum_repourls' needed?

Comment (lcarva) [inline tests/build_/test_arrangements.py]: No, it isn't. I thought for some reason that yum_repourls was required for detecting it's a custom base image build. Removed.

**Comment by lcarva** [comment]
Updated release notes.

**Pull request merged**